pkg/performanceprofile/controller/performanceprofile_controller.go

```diff
@@ -57,14 +58,20 @@ const finalizer = "foreground-deletion"
 // PerformanceProfileReconciler reconciles a PerformanceProfile object
 type PerformanceProfileReconciler struct {
 	client.Client
-	Scheme   *runtime.Scheme
-	Recorder record.EventRecorder
+	// Should only be used when on Hypershift deployment. In StandAlone OCP will be nil
```

Comment:
is the comment for ManagementClient only ?  if so then worth explicitly mentioning it